yellowbrick/classifier/top_features.py

```diff
@@ -0,0 +1,253 @@
+Visualize the top N features of classifier. This includes the N highest
+and the N lowest features.
+
+TODO:
+-   Docstrings
+-   Make quick method for producing plots for all classes using multiclass visualizer
+-   Add support for estimators with _feature_importance attributes
+-   Implement one-sided version that takes absolute value of coefs/features
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import numpy as np
+import warnings
+import matplotlib.pyplot as plt
+
+from .base import ClassificationScoreVisualizer
+from ..exceptions import YellowbrickValueError
+from ..style import color_palette
+
+##########################################################################
+## Top Features Visualizer
+##########################################################################
+
+class TopFeaturesVisualizer(ClassificationScoreVisualizer):
+    """
+    This takes the result of a classifier and visualizes
+    the top N positive and negative coefficients for a target class.
+    
+    Scikit-learn estimators store this information in two attributes:
+    coef_ and feature_importances_.
+    
+    Any classifier given to this visualizer must have coef_, 
+    otherwise we'll throw a "Estimator Not Implemented" warning.
+
+    Parameters
+    ----------
+    ax : matplotlib axes, default: None
+        The axes to plot the figure on. If None is passed in, the
+        current axes will be used (or generated if required).
+
+
+    N : int
+        The number of features to visualize among the positive and
+        negative coefficients.
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    Examples
+    --------
+    # Text Example
+    >>> from sklearn.svm import LinearSVC
+    >>> from yellowbrick.classifier import TopFeaturesVisualizer
+    >>> visualizer = TopFeaturesVisualizer(LinearSVC())
+    >>> visualizer.fit()
+    >>> visualizer.poof()
+    
+    # Numeric Example
+    >>> # numeric example here!!!!
+
+
+    Notes
+    -----
+    Text corpora need to be formatted as a vectorizer and 
+    the top features extracted before using the visualizer.
+    
+    data = corpus.data
+    target = corpus.target
+    vectorizer = CountVectorizer(stop_words='english')
+    vectorizer.fit(data)
+    X_train = vectorizer.transform(data)
+    features = vectorizer.get_feature_names()
+
+    """
+
+    def __init__(self, model, ax=None, N=20, features=None, **kwargs):
+        # TODO: Make sure model is a classifier; otherwise throw a warning.
+        # Check to see if there are N features; otherwise replace with count of features
+
+        super(TopFeaturesVisualizer, self).__init__(model=model, ax=ax, **kwargs)
+
+        self.N = N
+        self.features = features
+
+
+    def fit(self, X_train, y, **kwargs):
```

Comment:
Everything that you're doing here is done by the super class, so you don't need to specify a `fit()` method unless you're going to do something extra. (Also make sure to update your branch to take advantage of the super class features; let me know if you'd like me to do it)